src/main/scala/com/tapad/docker/DockerComposePlugin.scala

```diff
@@ -415,7 +416,7 @@ class DockerComposePluginLocal extends AutoPlugin with ComposeFile with DockerCo
     val requiresShutdown = getMatchingRunningInstance(newState, args).isEmpty
     val (preTestState, instance) = getTestPassInstance(newState, args)
 
-    val finalState = runTestPass(preTestState, args, instance)
+    val finalState = runTestPass(preTestState, args, instance, runVariablesForTestEnvTask(state))
```

Comment:
You should be able to move this call to occur inside of `runTestPass` instead of having to pass it in as a parameter and then do something like:

` val extraTestParams = runVariablesForTestEnvTask(state).map { case (k, v) => s"-D$k=$v" }`